packages/lib-sourcify/test/verification.spec.ts

```diff
@@ -1,1318 +0,0 @@
-        __dirname,
-        'sources',
-        'AbstractCreationBytecodeAttack',
-      );
-      const checkedContracts = await checkFilesWithMetadataFromContractFolder(
-        maliciousContractFolderPath,
-      );
-      const recompiled = await checkedContracts[0].recompile();
-      const match = {
-        address: contractAddress,
-        chainId: sourcifyChainHardhat.chainId.toString(),
-        runtimeMatch: null,
-        creationMatch: null,
-      };
-      const recompiledMetadata: Metadata = JSON.parse(recompiled.metadata);
-
-      const generateCreationCborAuxdataPositions = async () => {
-        if (!checkedContracts[0].creationBytecodeCborAuxdata) {
-          await checkedContracts[0].generateCborAuxdataPositions();
-        }
-        return checkedContracts[0].creationBytecodeCborAuxdata || {};
-      };
-      await matchWithCreationTx(
-        match,
-        recompiled.creationBytecode,
-        sourcifyChainHardhat,
-        contractAddress,
-        creatorTxHash,
-        recompiledMetadata,
-        generateCreationCborAuxdataPositions,
-        recompiled.creationLinkReferences,
-      );
-      expectMatch(match, null, contractAddress, undefined); // status is null
-    });
-
-    it('should successfully verify with matchWithCreationTx with creationTxHash', async () => {
-      const contractFolderPath = path.join(
-        __dirname,
-        'sources',
-        'WithImmutables',
-      );
-      const { contractAddress, txHash: creatorTxHash } =
-        await deployFromAbiAndBytecode(signer, contractFolderPath, ['12345']);
-
-      const checkedContracts =
-        await checkFilesWithMetadataFromContractFolder(contractFolderPath);
-      const recompiled = await checkedContracts[0].recompile();
-      const match = {
-        address: contractAddress,
-        chainId: sourcifyChainHardhat.chainId.toString(),
-        runtimeMatch: null,
-        creationMatch: null,
-      };
-      const recompiledMetadata: Metadata = JSON.parse(recompiled.metadata);
-      const generateCreationCborAuxdataPositions = async () => {
-        if (!checkedContracts[0].creationBytecodeCborAuxdata) {
-          await checkedContracts[0].generateCborAuxdataPositions();
-        }
-        return checkedContracts[0].creationBytecodeCborAuxdata || {};
-      };
-      await matchWithCreationTx(
-        match,
-        recompiled.creationBytecode,
-        sourcifyChainHardhat,
-        contractAddress,
-        creatorTxHash,
-        recompiledMetadata,
-        generateCreationCborAuxdataPositions,
-        recompiled.creationLinkReferences,
-      );
-      try {
-        expect(match.creationMatch).to.equal('perfect');
-        expect(match.address).to.equal(contractAddress);
-      } catch (e) {
-        console.log('Match: ', match);
-        throw e;
-      }
-    });
-
-    it('should find a perfect match for a contract that has a single metadata hash in the middle of the creation bytecode', async () => {
```

Comment:
here

Reply:
we have "should handle multiple auxdatas correctly"

Reply:
that should be fine then